Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -113,6 +113,11 @@ def append_path(key, path)
     self[key] = PATH.new(self[key]).append(path)
   end
 
+  sig { params(rustflags: String).void }
+  def append_to_rustflags(rustflags)
+    self["HOMEBREW_RUSTFLAGS"] = "#{self["HOMEBREW_RUSTFLAGS"]} #{rustflags}".strip
```

Comment:
```suggestion
    append("HOMEBREW_RUSTFLAGS", rustflags)
```
We already have the `#append` method for this.